backend/src/apiserver/client/subject_access_review_fake.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2020 Arrikto Inc.  All Rights Reserved.
+
+package client
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
+	authzv1 "k8s.io/api/authorization/v1"
+)
+
+type FakeSubjectAccessReviewClient struct {
+}
+
+func (FakeSubjectAccessReviewClient) Create(*authzv1.SubjectAccessReview) (*authzv1.SubjectAccessReview, error) {
+	return &authzv1.SubjectAccessReview{Status: authzv1.SubjectAccessReviewStatus{
+		Allowed:         true,
+		Denied:          false,
```

Comment:
When will denied be true, what does it mean?

Reply:
From the [documentation](https://godoc.org/k8s.io/api/authorization/v1#SubjectAccessReviewStatus):
```go
type SubjectAccessReviewStatus struct {
    // Allowed is required. True if the action would be allowed, false otherwise.
    Allowed bool `json:"allowed" protobuf:"varint,1,opt,name=allowed"`
    // Denied is optional. True if the action would be denied, otherwise
    // false. If both allowed is false and denied is false, then the
    // authorizer has no opinion on whether to authorize the action. Denied
    // may not be true if Allowed is true.
    // +optional
    Denied bool `json:"denied,omitempty" protobuf:"varint,4,opt,name=denied"`
    ...
}
```
So I believe we only care about the `Allowed` field to decide.
Maybe if we had multiple authorization mechanisms **and** we found out that `allowed == denied == false`, then we should proceed with the next mechanism.

Reply:
SGTM, thanks for the info!